images/jupyterhub/requirements.in

```diff
@@ -1,27 +1,2 @@
-## Authenticators
-jupyterhub-dummyauthenticator
-jupyterhub-firstuseauthenticator
-jupyterhub-hmacauthenticator
-jupyterhub-ldapauthenticator
-jupyterhub-ltiauthenticator
-nullauthenticator
-oauthenticator
-
-## Kubernetes spawner
-jupyterhub-kubespawner
-
-## Other optional dependencies for additional features
-psycopg2-binary  # postgres
-pycurl  # internal http requests handle more load with pycurl
-statsd  # statsd metrics collection (TODO: remove soon, since folks use prometheus)
-
-# The idle culler service
-jupyterhub-idle-culler
-
-## Useful tools
-
-# py-spy is useful for profiling running hubs
-py-spy
-
 ## IllumiDesk package
-https://github.com/IllumiDesk/illumidesk/archive/v0.9.1.zip
+https://github.com/IllumiDesk/illumidesk/archive/v1.0.0.zip
```

Comment:
Waiting on Illumidesk/Illumidesk code merge for the zip.
Should we also add jupyterhub-samlauthenticator since that's something we need to test?

Reply:
@Abhi94N we can add the `jupyterhub-samlauthenticator` if necessary in a separate PR since we need to test Auth0 with the standard `jupyterhub/oauthenticator` first.